readme.txt

```diff
@@ -1,37 +1,35 @@
+= How ImageEngine Works =
 
-* When a visitor requests a page, ImageEngine dynamically pulls images from your WordPress origin and optimizes them.
-* At the CDN edge, ImageEngine uses device detection to identify the requesting devices and browser characteristics.
-* Based on this information, ImageEngine will resize, compress and encode images so they are best suited to the end users' device.
-* The optimized image will then be delivered at the ImageEngine CDN edge. Subsequent requests are served instantly with cached assets stored on ImageEngine’s global CDN.
+This plugin rewrites your image URLs to include the ImageEngine Delivery Address you recieve when you sign up for an ImageEngine account. This rewrite will allow ImageEngine to access your original images, instantly optimize and convert them, and deliver via the ImageEngine CDN. After configuring and enabling the plug in, image are delivered this way:
 
-Users of [ImageEngine](https://imageengine.io/?utm_source=wordpress.org&utm_medium=page&utm_term=wp-imageengine&utm_campaign=wp-imageengine) can also configure the Directives for their assets to control image quality, automatic format conversion and automatic image resizing.
+* When a visitor requests an image, ImageEngine CDN servers use client hints or device detection to identify the requesting devices and browser characteristics.
+* Based on the browser characteristics, ImageEngine will resize, compress and convert images to WebP, JPEG 2000, or AVIF.
+* The optimized image is  delivered from the ImageEngine CDN edge. Subsequent requests are served instantly with WebP, JPEG 2000, or AVIF images stored on ImageEngine’s global CDN.
 
 = What Makes ImageEngine Better Than Other CDNs or Digital Asset Management Platforms? =
 
-* Delivers optimized images 30% faster than other CDNs or Digital Asset Management platforms.
+* Delivers optimized WebP, JPEG 2000, or AVIF images 30% faster than other CDNs or Digital Asset Management platforms.
 * Achieves up to 80% image payload reduction with no perceptible change in quality.
 * Simple to install. Easy to test your configuration before enabling it. No need to move or upload images.
-* Only CDN with true device-aware edge servers to drive superior, fine-tuned optimization.
-* Automatic image optimization of JPG, PNG, GIF, SVG, BMP, TIF into next generation formats like WebP, JPEG 2000, aWebP, or MP4. You can also safely serve non-image files through ImageEngine.
+* Only CDN with true device-aware edge servers to drive superior, fine-tuned image optimization.
+* Automatic image optimization of JPG, PNG, GIF, SVG, BMP, TIF into next generation formats like WebP, JPEG 2000, AVIF, Animated WebP, or MP4. You can also safely serve non-image files through ImageEngine.
```

Comment:
```suggestion
* Automatic image optimization of JPG, PNG, GIF, SVG, BMP, TIFF into next-generation formats like WebP, JPEG 2000, AVIF, Animated WebP, or MP4. You can also safely serve non-image content like CSS, Javascript, Fonts and any other files through ImageEngine and they will be served using the cutting-edge Brotli compression, decreasing Total Blocking Time and Time to Interactive.
```

Reply:
Maybe we should put a note somewhere that this plugin "dramatically reduces that size of heavy WordPress themes and optimizes all of their Javascript, CSS and Fonts using Brotli and HTTP/2".